docs/Overview.md

```diff
@@ -414,11 +413,11 @@ def __call__(self, hidden_states: NamedArray, layer_idx, inference: bool = True,
 If you're not used to the `tril`-as-a-mask trick, it's a way to create a causal mask so that the attention scores
 for a query can only attend to the past. The `tril` function creates a lower triangular matrix. It's equivalent to:
 ```python
-causal_mask = jnp.zeros(seq_len, key_seq_len)
+causal_mask = jnp.zeros((seq_len, key_seq_len))
 for i in range(seq_len):
     for j in range(key_seq_len):
-        if j >= i:
```

Comment:
Since it is lower triangular matrix, I believe the sign should be `<=` instead of `>=`.

With `>= i:`, we get: 

```
[[1. 1. 1.]
 [0. 1. 1.]
 [0. 0. 1.]]
```

Instead, with `<= i`, we get:
```
[[1. 0. 0.]
 [1. 1. 0.]
 [1. 1. 1.]]
```